### PR TITLE
BAEL-3341 A Guide to Hibernate Types Project

### PR DESCRIPTION
**Pull request by andrebrowne**

### CODE REVIEW

File: architecture/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Relevant articles
+
+- [A Quick and Practical Example of Hexagonal Architecture in Java](https://www.baeldung.com/a-quick-and-practical-example-of-hexagonal-architecture-in-java-3/)
```
Comment (rpvilao) [inline architecture/README.md:3]: remove, we do this in the end after the article is published

**Comment by andrebrowne** [comment]
@rpvilao I see the problem here. My test article commits for Hexagonal Architecture are in this fork. I'll remove them.

File: architecture/README.md (hunk 1 above)
Comment (andrebrowne) [reply architecture/README.md:3]: Removed in new PR. We should update _baeldung - for authors - 4 - implementation code_ to say that the README.md should be updated after the article is published. Should I update the README.md file in the hibernate types sub-module.

File: persistence-modules/hibernate-types/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+version: '3.2'
```
Comment (rpvilao) [inline persistence-modules/hibernate-types/docker-compose.yml:1]: remove

**Pull request closed without merging**